src/validation_workflow/validation_args.py

```diff
@@ -43,25 +41,33 @@ def __init__(self) -> None:
             "--platform",
             type=str,
             choices=self.SUPPORTED_PLATFORMS,
-            help="Platform to validate.",
+            help="(optional) Platform to validate.",
             default="linux"
         )
         parser.add_argument(
             "--stgosbuild",
             type=str,
             required=False,
-            help="The opensearchstaging OpenSearch image build number if required, for example : 6039\n",
+            help="(optional) The opensearchstaging OpenSearch image build number if required, for example : 6039\n",
             default="",
             dest="stgosbuild",
         )
         parser.add_argument(
             "--stgosdbuild",
             type=str,
             required=False,
-            help="The opensearchstaging OpenSearchDashboard image build number if required, for example : 4104\n",
+            help="(optional) The opensearchstaging OpenSearchDashboard image build number if required, for example : 4104\n",
             default="",
             dest="stgosdbuild",
         )
+        parser.add_argument(
+            "--dockersource",
```

Comment:
rewording recommendation: `--docker-source`

Reply:
Thanks, @gaiksaya , sounds good, let us update it on next commit.